vhdbuilder/packer/install-dependencies.sh

```diff
@@ -165,7 +165,7 @@ done
 
 installAndConfigureArtifactStreaming() {
   # download acr-mirror proxy
-  MIRROR_PROXY_VERSION='8'
+  MIRROR_PROXY_VERSION='9'
```

Comment:
can we automate this? like how we automated the go version for automation? might not be as easy but just curious since we already update this some days ago

Reply:
Current phase involves more debugging/coordination with ACR so we want to pick it up manually. Based on stability and testing later, we can automate it.